contracts/ocr2/src/state.rs

```diff
@@ -35,12 +35,17 @@ pub struct Validator {
     pub gas_limit: u64,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Default, Debug, PartialEq, JsonSchema)]
 pub struct Billing {
     /// Should match https://fcd.terra.dev/v1/txs/gas_prices
     /// NOTE: needs to be scaled to the same amount of decimals places as LINK token
```

Comment:
Which configs here need to be scaled, please be a little more specific here.

Can we do a quick and safe sanity check on these configs on the input so we revert if:

- values are really low (not scaled properly)?
- values are under contract defaults, does it make sense to go under defaults?

Reply:
Extracted as GH Issue:

- https://github.com/smartcontractkit/chainlink-terra/issues/58